packages/ts/react-crud/src/autoform.tsx

```diff
@@ -76,8 +76,14 @@ export function ExperimentalAutoForm<TItem>({
           <AutoFormField key={propertyInfo.name} propertyInfo={propertyInfo} form={form} disabled={disabled} />
         ))}
       {formError ? <div style={{ color: 'var(--lumo-error-color)' }}>{formError}</div> : <></>}
-      <HorizontalLayout style={{ marginTop: 'var(--lumo-space-m)' }}>
+      <HorizontalLayout theme="spacing" style={{ marginTop: 'var(--lumo-space-m)', alignSelf: 'flex-end' }}>
+        {form.dirty ? (
+          <Button theme="tertiary" onClick={() => form.reset()}>
+            Discard
+          </Button>
+        ) : null}
```

Comment:
I think the current behavior makes sense as long as it's only a discard button. One thing that I was thinking about it whether it should rather be a "cancel" button that would actually deselect the item and close the form (which is what the CRUD component has). IMO this is a UX issue, and as mentioned in the Daily yesterday we should consult with an expert on what the UX of the auto crud should be instead of trying to figure this out ourselves. For now I'd suggest to just add the feature and clean up the UX in a second pass.